src/themes/teams/index.ts

```diff
@@ -34,11 +34,36 @@ Object.keys(fontIcons).forEach(iconName => {
   icons[iconName] = declareFontBased(fontIcons[iconName])
 })
 
+const animations = {
+  spinner: {
+    keyframe: () => ({
```

Comment:
For now, we won't accept params to keyframes (to prevent rendering new animations on prop changes).   Instead, just accept an object here and use `callable()` on it internally.